src/ng-select/ng-select.types.ts

```diff
@@ -22,6 +22,7 @@ export enum KeyCode {
 }
 
 export interface NgSelectConfig {
+    openOnEnter?: boolean;
```

Comment:
set it to true by default

Reply:
Set it to true by default in the module configuration.